### PR TITLE
Fixing include composer autoloader

### PR DESCRIPTION
**Pull request by TomKlotzPro**
| Q                 | A
| ----------------- | ----------
| Bug fix?          | no
| New feature?      | no   
| BC breaks?        | no     
| Related Issue     | https://github.com/algolia/algoliasearch-client-php/pull/486
| Need Doc update   | no

## Describe your change

Including composer autoloader.

> This fix is related to this issue : https://github.com/algolia/algoliasearch-client-php/pull/486 

## What problem is this fixing?

 Check if [GuzzleClient](https://packagist.org/packages/guzzlehttp/guzzle)  is available.

### CODE REVIEW

**Comment by TomKlotzPro** [comment]
need a review @nunomaduro

File: bin/algolia-doctor [hunk 1]
```diff
@@ -1,6 +1,15 @@
 #!/usr/bin/env php
 <?php
 
+// This kind of sucks.. but we need the composer autoloader, in order to fully check for GuzzleHttp
```
Comment (nunomaduro) [inline bin/algolia-doctor]: I think you can remove this comment.

File: bin/algolia-doctor [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+if (file_exists(__DIR__ . '/../../../autoload.php')) {
+    include __DIR__ . '/../../../autoload.php';
+} else if (file_exists(__DIR__ . '/../autoload.php')) {
+    // fallback to the local non-composer autoloader which seems to fail to check for Guzzle
```
Comment (nunomaduro) [inline bin/algolia-doctor]: I think you can remove this comment.

File: bin/algolia-doctor (hunk 1 above)
Comment (TomKlotzPro) [reply bin/algolia-doctor]: 👍

File: bin/algolia-doctor (hunk 2 above)
Comment (TomKlotzPro) [reply bin/algolia-doctor]: 👍

**Comment by TomKlotzPro** [comment]
need review @nunomaduro

**Pull request merged**